common/src/main/java/org/tron/core/Constant.java

```diff
@@ -92,8 +92,10 @@
   public static final String NODE_P2P_VERSION = "node.p2p.version";
   public static final String NODE_RPC_PORT = "node.rpc.port";
   public static final String NODE_RPC_SOLIDITY_PORT = "node.rpc.solidityPort";
+  public static final String NODE_RPC_PBFT_PORT = "node.rpc.PBFTPort";
   public static final String NODE_HTTP_FULLNODE_PORT = "node.http.fullNodePort";
   public static final String NODE_HTTP_SOLIDITY_PORT = "node.http.solidityPort";
+  public static final String NODE_HTTP_PBFT_PORT = "node.http.PBFTPort";
```

Comment:
Be consistent with other configuration items.
Since you have node.max**Http**ConnectNumber already, `node.http.PbftPort` is preferred.